test/validator.js

```diff
@@ -10,54 +10,73 @@ describe('ALF Spec', function () {
     validate({}, function (err, valid) {
       valid.should.be.false;
 
-      err[0].should.have.property('message');
-      err[0].message.should.equal('is required');
-    });
+      err[0].should.have.property('field').and.equal('data.serviceToken');
+      err[0].should.have.property('message').and.equal('is required');
+
+      err[1].should.have.property('field').and.equal('data.har');
+      err[1].should.have.property('message').and.equal('is required');
 
-    done();
+      done();
+    });
   });
 
   it('should fail with empty array', function (done) {
     validate([], function (err, valid) {
       valid.should.be.false;
 
-      err[0].should.have.property('message');
-      err[0].message.should.equal('is the wrong type');
-    });
+      err[0].should.have.property('field').and.equal('data');
+      err[0].should.have.property('message').and.equal('is the wrong type');
 
-    done();
+      done();
+    });
   });
 
   it('should fail with undefined', function (done) {
     validate(undefined, function (err, valid) {
       valid.should.be.false;
-    });
 
-    done();
+      err[0].should.have.property('field').and.equal('data');
```

Comment:
none, since `is-my-json-valid` does not clear its internal pointers ... just remove, this case is covered with the `valid === false` check